torch/distributed/run.py

```diff
@@ -823,12 +862,22 @@ def config_from_args(args) -> tuple[LaunchConfig, Union[Callable, str], list[str
     cmd: Union[Callable, str]
     cmd_args = []
     use_env = get_use_env(args)
+    numa_cmd = None
+    py_executable = os.getenv("PYTHON_EXEC", sys.executable)
+    if args.numa_binding:
+        numa_cmd = update_with_numa_binding_pytorch(args.numa_binding)
```

Comment:
Can we implement this at the elastic agent level? Putting this logic here means only CLI users can get numa control and not via the programmatic API